cybered/hashing/templates/hashing/conclusion.html

```diff
@@ -0,0 +1,38 @@
+<!doctype HTML>
+<html>
+{% include "includes/head.html" %}
+
+<body>
+    {% include "includes/header.html" %}
+    <div class="container">
+        <h3>Conclusion</h3>
+
+        <p>
+            You've completed the CyberEd module on hashing! We started by motivating the need for message
+            digests with a story about Alice, Bob, and Chuck, and then learned about what a message digest is,
+            how a hash function makes one, and what makes a hash function good.
```

Comment:
I think we should summarize the important properties of a hash function again. I'm thinking for a module on passwords it could come in handy for them to know
* Hash functions are one-way
* Small changes in the input produce large changes in the output
* Variable-length inputs produce fixed-length hashes
* It's hard to produce a collision